src/components/Hero/hero.scss

```diff
@@ -2,11 +2,27 @@
 
 .top {
   position: relative;
-  min-height: 440px;
   min-width: 100%;
+}
+
+.top h1 {
```

Comment:
Should we ditch the `<Heading1>` usage here if we are over-writing a lot of stuff?